e2e/test-registration-data/test-registrations-westeros-20.csv

```diff
@@ -1,21 +1,21 @@
-05dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,ironBank,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-06dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,ironBank,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-07dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,ironBank,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-08dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,ironBank,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-09dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,ironBank,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-10dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-11dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-12dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-13dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-14dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238876,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-15dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238856,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-16dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-17dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-18dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
-19dc9451-1273-484c-b2e8-ae21b51a96ab,en,,Test,succeed,14155238886,gringotts,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
+referenceId,preferredLanguage,maxPayments,fullName,phoneNumber,programFinancialServiceProviderConfigurationName,whatsappPhoneNumber,house,dragon,knowsNothing,motto,personalId,date,accountId,openAnswer,fixedChoice,healthArea
+00dc9451-1273-484c-b2e8-ae21b51a96ab,en,,John Snow,14155238886,ironBank,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
+01dc9451-1273-484c-b2e8-ae21b51a96ab,en,,John Snow,14155238886,ironBank,14155238886,stark,0,1,Winter is coming,id1,30-06-1994,1,agreed,yes,HealthAreaName
```

Comment:
No unique names for each?

(I haven't seen GoT, so I don't know if there are 20 people 'available to use' (i.e. still alive). ;) )

Reply:
That would be better, but leaving it for a next PR ;)